src/styles/index.scss

```diff
@@ -1,3 +1,123 @@
+$main--text--color: #060b35;
+$secondary--text--color: #616070;
+$card--border--color: #f3f3f3;
+$main--button--color: #00acdc;
+$secondary--button--color: #fff;
+
+* {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  color: $main--text--color;
+  line-height: 18px;
+
   margin: 0;
 }
+
+.product {
+  // display: flex;
+  // flex-direction: column;
+  // box-sizing: border-box;
```

Comment:
Create separate files for BEM blocks (product and card)
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/c0e623f4-6850-43bb-801b-9c8dd5f3dfb5)